packages/core/app/layout.tsx

```diff
@@ -78,11 +80,13 @@ export default async function RootLayout({
       <AnalyticsHandler />
 
       <html>
-        <head>
-          {!process.env.DISABLE_3P_SCRIPTS && <ThirdPartyScripts />}
-          <WebFonts />
-        </head>
-        <body className="theme">
```

Comment:
don't we need this `theme` className?

I think we still need: https://developers.vtex.com/docs/guides/faststore/themes-overview
c.c. @hellofanny @renatamottam

Reply:
Oh this was my mistake, I forgot to move this back along with the `DefaultFont.classname` 🤡  I'll fix this